media-libs/libffado/libffado-2.3.0.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+[[ "${PV}" = "9999" ]] && inherit subversion
+inherit eutils python-single-r1 scons-utils toolchain-funcs udev multilib-minimal
```

Comment:
I think the answer is "probably nobody cares but getting rid of it is hard". The JACK client library definitely needs to be built with multilib support. That doesn't depend on ffado. But the JACK server is also a library and you can run a JACK server in-process. Does anyone actually do that? Do random things link to that library that might pull it in for 32-bit apps? Maybe we could just drop the drivers since surely nobody actually runs a 32-bit in process server on a 64-bit OS?

I don't know the answers to these questions, but it would involve messing with the JACK ebuild quite a bit more, and... that's probably more effort than just supporting multilib here.